### PR TITLE
Webc 1566 toasts

### PR DESCRIPTION
**Pull request by ainvoner**

### CODE REVIEW

File: packages/ui/src/components/kitchen-sink-adapter/kitchen-sink-adapter.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { KitchenSinkExpandModes, KitchenSinkPositions } from "../../kitchenSinkI
 
 export interface KitchenSinkAdapterProps {
     updateSidePanelMode: (position: SidePanelPositions, sidePanelMode: SidePanelModes) => void;
+    sidePanelsModes?: any;
```
[the remark this answers is not in the document]
Comment (ainvoner) [reply packages/ui/src/components/kitchen-sink-adapter/kitchen-sink-adapter.tsx]: changed to SidePanelModes

File: packages/ui/src/kitchenSinkItemData.ts [hunk 2]
```diff
@@ -5,8 +5,9 @@ export interface KitchenSinkContentRendererProps {
 }
 
 export enum KitchenSinkExpandModes {
-    AlongSideTheVideo,
-    OverTheVideo
+    AlongSideTheVideo = "ALONG_SIDE_THE_VIDEO",
```
[the remark this answers is not in the document]
Comment (ainvoner) [reply packages/ui/src/kitchenSinkItemData.ts]: yes: https://github.com/kaltura/playkit-js-ui/blob/FEC-8696-contrib-presets/src/reducers/shell.js

File: packages/ui/src/components/toasts-container/toastsContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+export interface ToastsContainerProps {
+    toasts: ToastProps[];
+}
+
+export class ToastsContainer extends Component<ToastsContainerProps> {
+    render() {
+        return (
+            <div className={styles.toastsContainer}>
+                {" "}
```
[the remark this answers is not in the document]
Comment (ainvoner) [reply packages/ui/src/components/toasts-container/toastsContainer.tsx]: it is a very good question :) just a mistake

File: packages/ui/src/components/toasts-container/toastsContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+                {this.props.toasts.map(toast => {
+                    return (
+                        <div className={styles.toastRow}>
```
[the remark this answers is not in the document]
Comment (ainvoner) [reply packages/ui/src/components/toasts-container/toastsContainer.tsx]: done

File: packages/ui/src/toastsManager.tsx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+import { ToastProps } from "./components/toast/toast";
+import { ToastsContainer } from "./components/toasts-container/toastsContainer";
+import { h } from "preact";
+
+export interface ToastsManagerOptions {
+    overlayManager: OverlayManager;
+}
+
+export enum ToastSeverity {
+    INFO = "INFO",
```
[the remark this answers is not in the document]
Comment (ainvoner) [reply packages/ui/src/toastsManager.tsx]: done

**Comment by esakal** [comment]
@ainvoner feel free to merge tomorrow and verify that master works

**Pull request merged**